README.md

```diff
@@ -1,51 +1,98 @@
 # Kik Bot API #
-Use this library to develop bots for [Kik Messenger](https://www.kik.com) that are essentially automated humans.
+The Kik Bot API is a Python library developed to automate interactions on [Kik Messenger](https://www.kik.com) that are essentially automated humans.
```

Comment:
"Kik Bot API" could be confused with the [official one](https://www.kik.com/developers/), I think we need to make it clear it's different